pkg/version/version.go

```diff
@@ -17,12 +17,12 @@ var (
 	//
 	// Version must conform to the format expected by github.com/hashicorp/go-version
 	// for tests to work.
-	Version = "1.2.0"
+	Version = "1.2.1"
 
 	// A pre-release marker for the version. If this is "" (empty string)
 	// then it means that it is a final release. Otherwise, this is a pre-release
 	// such as "dev" (in development), "beta", "rc1", etc.
-	VersionPrerelease = "rc1"
+	VersionPrerelease = ""
```

Comment:
I think we want `dev` for testing unless we are releasing this week.
```suggestion
	VersionPrerelease = "dev"
```